templates/islandora-paged-tei-seadragon-viewer.tpl.php

```diff
@@ -0,0 +1,47 @@
+<?php
+/**
+ * @file
+ * Template file for the paged TEI seadragon viewer.
+ */
+?>
+<?php if ($tei_pages_populated): ?>
```

Comment:
Need to check with `<?php if (isset($tei_pages_populated) && $tei_pages_populated): ?>`